Newsletter2Go/Export/Model/Config/Source/IntegrationCreateButton.php

```diff
@@ -39,13 +38,6 @@ protected function _getElementHtml(AbstractElement $element)
             '<language>' => $languageCode[0],
         ];
         $url = str_replace(array_keys($replacements), array_values($replacements), self::N2G_CONNECT_URL);
-        // add callback url parameter
-        $url .= '&callback=' . $shopUrl . 'n2gocallback/Callback/Index';
```

Comment:
These changes already exists above, currently they caused an internal error in the API side.